### DISCUSSION TITLE
Intermittent styling issue with custom syntax highlighter

### DISCUSSION THREAD

**Discussion by RobertCraigie** [squidfunk/mkdocs-material#3728] [Q&A] [answered]
Hi!

I'm using Shiki as a syntax highlighter for my documentation as Pygments doesn't support the Prisma Schema language yet.

The script I'm using to highlight the code blocks can be found here: https://github.com/RobertCraigie/prisma-client-py/blob/main/docs/scripts/highlight.js

Occasionally the following issue can happen: https://anand2312.tech/img/uploads/firefox_7VPtfoICab.gif

Documentation can be found here: https://prisma-client-py.readthedocs.io/en/stable/

Does anyone know what could cause this?

**Comment by squidfunk** [comment, marked as answer]
Pygments is the only syntax highlighter supported by Material for MkDocs. For everything else, please consult the authors of the plugin or extension you are using to do syntax highlighting. We can't offer support for anything else.
